lua/core/options.lua

```diff
@@ -31,6 +31,8 @@ opt.ruler = options.ruler
 -- disable nvim intro
 opt.shortmess:append "sI"
 
+
```

Comment:
Could you edit your PR commit using: https://nvchad.netlify.app/contribute#how-to-remove-or-edit-commits-from-your-pr

And remove these two empty lines please! :)

Reply:
I see